R/selections.R

```diff
@@ -106,27 +107,30 @@ eval_select_recipes <- function(quos, data, info) {
 
   expr <- expr(c(!!!quos))
 
-  # FIXME: Ideally this is `FALSE`, but empty selections incorrectly throw an
+  # FIXME: Ideally this is `FALSE` for strict selection,
+  # but empty selections incorrectly throw an
   # error when this is false due to the following bug:
   # https://github.com/r-lib/tidyselect/issues/221
-  allow_rename <- TRUE
+  # Once it's fixed, remove this and pass allow_rename to
+  # tidyselect::eval_select().
+  allow_rename_ts <- TRUE
```

Comment:
```suggestion
  allow_rename_compat <- TRUE
```

Suffixing with `compat`ibility is a nicer indicator to me of what this is for, since now we have a real `allow_rename` argument

Reply:
agree - I used _ts for tidyselect but _compat is clearer